components/rebrandly/rebrandly.app.mjs

```diff
@@ -1,11 +1,101 @@
+import { axios } from "@pipedream/platform";
+
 export default {
   type: "app",
   app: "rebrandly",
-  propDefinitions: {},
+  propDefinitions: {
+    domain: {
+      label: "Domain",
+      description: "The domain to be used",
+      type: "string",
+      async options({ prevContext }) {
+        const domains = await this.getDomains({
+          params: {
+            last: prevContext.lastId,
+            orderBy: "createdAt",
+            orderDir: "asc",
+          },
+        });
+
+        return {
+          options: domains.map((domain) => ({
+            label: domain.fullName,
+            value: domain.id,
+          })),
+          context: {
+            lastId: domains[domains.length].id,
```

Comment:
Shouldn't this be `domains[domains.length - 1].id` ?
And if so, a check if `domains.length > 0` would be necessary to not make it `undefined`

Reply:
You are right, sorry!